spanner/api/Spanner.Samples/DeleteDataAsync.cs

```diff
@@ -0,0 +1,171 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_delete_data]
+
+using Google.Cloud.Spanner.Data;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+public class DeleteDataAsyncSample
+{
+    public class Album
+    {
+        public int SingerId { get; set; }
+        public int AlbumId { get; set; }
+        public string AlbumTitle { get; set; }
+    }
+
+    public class Singer
+    {
+        public int SingerId { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+    }
+
+    public async Task<int> DeleteDataAsync(string projectId, string instanceId, string databaseId)
+    {
+        string connectionString = $"Data Source=projects/{projectId}/instances/{instanceId}/databases/{databaseId}";
+
+        // Create Sample Tables.
+        using (var connection = new SpannerConnection(connectionString))
```

Comment:
I'm not sure about this implementation. The [samples](https://cloud.google.com/spanner/docs/modify-mutation-api#deleting_rows_in_a_table) in other languages are very succinct, with all of the functionality fitting into few lines of code. My original C# implementation was a bit longer. But this sample is now almost 200 lines long and I don't think that's very readable.

Is there a better way to set it up?

Reply:
Remove all the Table creation and insert data logic from sample Now sample only contains Delete operation, database and table creation logic move to Sample Test.
Now Sample is same as other language.